src/WebJobs.Script/Description/DotNet/CSharp/CSharpCompilation.cs

```diff
@@ -29,7 +29,8 @@ public CSharpCompilation(Compilation compilation)
 
         public ImmutableArray<Diagnostic> GetDiagnostics()
         {
-            return _compilation.WithAnalyzers(GetAnalyzers()).GetAllDiagnosticsAsync().Result;
+            var diagnostics = _compilation.WithAnalyzers(GetAnalyzers()).GetAllDiagnosticsAsync().Result;
```

Comment:
GetAllDiagnostics gets all the analyzer diagnostics and errors. It's just poorly named.